.github/workflows/Tagged.yaml

```diff
@@ -0,0 +1,157 @@
+name: Tagged build
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+    paths-ignore:
+      - '.github/workflows/Untagged.yaml'
+  workflow_dispatch:
+
+env:
+  Configuration: Release
+  DSOALRepo: kcat/dsoal
+  DSOALBranch: ${{github.ref_name}}
+  OpenALSoftRepo: kcat/openal-soft
+  OpenALSoftBranch: master
+
+permissions:
+  contents: write
+
+jobs:
+  Build:
+    name: ${{matrix.config.name}}
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        config:
+        - {
+            name: "Win32",
+            platform: "Win32"
+          }
+        - {
+            name: "Win64",
+            platform: "x64"
+          }
+    steps:
+    
+    - name: Clone DSOAL
+      run: |
+        git clone --branch ${{env.DSOALBranch}} https://github.com/${{env.DSOALRepo}}.git .
```

Comment:
Maybe it's because of it being in your own branch, but it looks weird to be cloning the repository the workflow is acting on. Is this needed?

Reply:
Yeah, it replaces `actions/checkout` and it was mostly use your repo's commit details, particularly count, for the nightly version like [DSOAL_r444](http://vaporeon.io/hosted/dsoal-builds/dsoal-r444.7z)... instead of DSOAL_r464... which includes the extra commits on my fork, though for the Tagged workflow, I also used it to clone my [(2nd) fork](https://github.com/BinauralAudio/dsoal/actions/workflows/Tagged.yaml) to test the [tag-triggered workflow](https://github.com/ThreeDeeJay/dsoal/blob/f1f8d71de8cd68bab8c3d6473aae6794a9e03d24/.github/workflows/Tagged.yaml#L5) since there's none upstream.

Anyhow, I now realize that I should use `${{github.ref_name}}` instead so the repo name isn't hard-coded.